wiki/Performance_troubleshooting/de.md

```diff
@@ -43,70 +39,70 @@ Während des Spielens friert alles lange genug ein, sodass das Spielerlebnis bee
+- Versuche, das FPS-Limit zu ändern und schaue, ob es hilft.
 - Sorge dafür, dass osu! im exklusiven Vollbildmodus (nicht im randlosen Modus) läuft. Das wird die Leistung immer verbessern.
 - Stelle sicher, dass du den [Spielmodus](https://www.giga.de/downloads/windows-10/tipps/windows-10-game-mode-was-ist-das/) aktiviert hast. Er erlaubt Windows die Bereitstellung dedizierter Kerne und Ressourcen für osu!.
-- Öffne den Task-Manager und ordne die Liste (unter Details) nach CPU. Schaue nach hohen CPU-Auslastungen während Perioden mit niedriger Bildrate, zum Beispiel durch einen Viren- oder Malware-Scanner im Hintergrund. Du kannst osu! im Fenstermodus laufen lassen oder den Task-Manager auf einem zweiten Bildschirm platzieren, um das zu diagnostizieren, während du selbst spielst oder eine Beatmap mit der Mod Auto ausführst.
+- Öffne den Task-Manager und ordne die Liste (unter Details) nach CPU. Schaue nach hohen CPU-Auslastungen während Perioden mit niedriger Framerate, zum Beispiel durch einen Viren- oder Malware-Scanner im Hintergrund. Du kannst osu! im Fenstermodus laufen lassen oder den Task-Manager auf einem zweiten Bildschirm platzieren, um das zu diagnostizieren, während du selbst spielst oder eine Beatmap mit der Mod Auto ausführst.
 
 ## FPS-Limits
 
-osu! bietet mehrere Auswahlmöglichkeiten von FPS-Limits an. Jede davon hat seine Vor- und Nachteile. Bedenke, dass osu! die Bildwiederholungsrate in Menüs unabhängig von dieser Einstellung limitiert, um die CPU-Auslastung zu schonen.
+osu! bietet mehrere FPS-Limits zur Auswahl an. Jedes davon hat Vor- und Nachteile. Bedenke, dass osu! die Framerate in Menüs unabhängig von dieser Einstellung limitiert, um die CPU-Auslastung zu verringern.
 
 ### VSync
 
-Dieses FPS-Limit verlässt sich auf einen vom Treiber bereitgestellten vertikalen Synchronisierungsmechanismus. Es garantiert, dass Screen Tearing nicht auftritt, aber fügt 1 bis 2 Frames an Latenzzeit aufgrund der Zwischenspeicherung von Bildern vor dem Anzeigen hinzu.
+Dieses FPS-Limit verlässt sich auf einen vom Treiber bereitgestellten vertikalen Synchronisierungsmechanismus. Es garantiert, dass kein Screen Tearing auftritt, fügt aber 1 bis 2 Frames an Latenz aufgrund der Zwischenspeicherung von Bildern vor dem Anzeigen hinzu.
 
 Allgemein wird das nicht für Bildschirme mit 60 Hz empfohlen. Es kann bei Wiederholungsraten ab 120 Hz nützlich sein, aber mit GSync- und FreeSync-Bildschirmen ist es nicht erforderlich, VSync zur Verhinderung von Screen Tearing zu benutzen.
 
 ### Power Saving
 
-Dieses FPS-Limit zielt darauf ab, Strom zu sparen und gleichzeitig ein gleichmäßiges Spielerlebnis zu bieten. Es läuft mit 2-facher Bildwiederholungsrate und versucht, ein einheitliches Frame-Pacing beizubehalten.
+Dieses FPS-Limit zielt darauf ab, Strom zu sparen und gleichzeitig ein gleichmäßiges Spielerlebnis zu bieten. Es läuft mit 2-facher Bildwiederholungsrate und versucht, gleichmäßige Frametimes beizubehalten.
```

Comment:
```suggestion
Dieses FPS-Limit zielt darauf ab, Strom zu sparen und gleichzeitig ein gleichmäßiges Spielerlebnis zu bieten. Es läuft mit 2-facher Framerate und versucht, gleichmäßige Frametimes beizubehalten.
```

Reply:
In these sentences, Bildwiederholungsrate is the intended meaning (display refresh rate)